src/main/java/com/gmail/nossr50/util/ItemUtils.java

```diff
@@ -147,18 +149,29 @@ public static boolean hasItemIncludingOffHand(Player player, Material material)
      * @param amount   Amount of the material to remove
      */
     public static void removeItemIncludingOffHand(@NotNull Player player, @NotNull Material material, int amount) {
-        // Checks main inventory / item bar
-        if (player.getInventory().contains(material)) {
-            player.getInventory().removeItem(new ItemStack(material, amount));
```

Comment:
Could you please explain why you had to replace lines 150-153? It does not seem obvious why the code you've replaced it with would do anything differently.